src/python/test/test_dxclient.py

```diff
@@ -7489,6 +7489,43 @@ def test_get_app_omit_resources(self):
                     break
             self.assertTrue(seenResources)
 
+    @unittest.skipUnless(testutil.TEST_ENV, 'skipping test that would clobber your local environment')
+    @unittest.skipUnless(testutil.TEST_ISOLATED_ENV, 'skipping test that would create apps')
+    @unittest.skipUnless(testutil.TEST_MULTIPLE_USERS, 'skipping test that would require another user')
+    def test_uninstall_app(self):
+        second = json.loads(os.environ['DXTEST_SECOND_USER'])
+        second_user_id = second['user']
+        authorized_users = [second_user_id]
+        name = 'uninstall_test'
+        app_spec = {
+            "name": name,
+            "title": name,
+            "dxapi": "1.0.0",
+            "runSpec": {"file": "code.py", "interpreter": "python2.7"},
+            "inputSpec": [{"name": "in1", "class": "file"}],
+            "outputSpec": [{"name": "out1", "class": "file"}],
```

Comment:
From @ptn24:
Are inputSpec and outputSpec necessary? That is, we can specify empty lists for the input and output specifications, respectively. In general, I prefer to keep unnecessary information out of the tests to improve readability and maintainability.